iris_grib/tests/unit/save_rules/__init__.py

```diff
@@ -14,22 +14,26 @@
 #
 # You should have received a copy of the GNU Lesser General Public License
 # along with iris-grib.  If not, see <http://www.gnu.org/licenses/>.
-"""Unit tests for the :mod:`iris.fileformats.grib.grib_save_rules` module."""
+"""Unit tests for the :mod:`iris_grib.grib_save_rules` module."""
 
 from __future__ import (absolute_import, division, print_function)
 from six.moves import (filter, input, map, range, zip)  # noqa
 
-import iris.tests as tests
+# Import iris_grib.tests first so that some things can be initialised before
+# importing anything else.
+import iris_grib.tests as tests
+
+import numpy as np
 
 import iris
 from iris.fileformats.pp import EARTH_RADIUS as PP_DEFAULT_EARTH_RADIUS
-from iris.tests import mock
-import numpy as np
+
+from iris_grib.tests import mock
```

Comment:
I've used the same pattern here as in `iris` ... nothing new really.

Happy to refactor everything if you feel strongly enough ...